strings/en_US/messages.po

```diff
@@ -3588,3 +3582,342 @@ msgstr "Manage Users"
+
+msgid "Youth Mentors (under 18)"
+msgstr "Youth Mentors (under 18)"
+
+msgid "Parents of attendees"
+msgstr "Parents of attendees"
+
+msgid "All global Dojos follow a common ethos and principles, such as always being free for children, and encouraging them to work together."
+msgstr "All global Dojos follow a common ethos and principles, such as always being free for children, and encouraging them to work together."
+
+msgid "All of these principles are  documented in the CoderDojo Charter."
+msgstr "All of these principles are  documented in the CoderDojo Charter."
+
+msgid "To become a verified Dojo you must agree to follow all principles in the CoderDojo Charter."
+msgstr "To become a verified Dojo you must agree to follow all principles in the CoderDojo Charter."
+
+msgid "It’s important that, as part of the CoderDojo community, you and your volunteers understand the CoderDojo Ethos."
+msgstr "It’s important that, as part of the CoderDojo community, you and your volunteers understand the CoderDojo Ethos."
+
+msgid "To make it easier to understand and implement in your Dojo we have created a short E-Learning module <a target=\"_blank\" href=\"/dashboard/profile/lms\">which you can find here.</a>"
+msgstr "To make it easier to understand and implement in your Dojo we have created a short E-Learning module <a target=\"_blank\" href=\"/dashboard/profile/lms\">which you can find here.</a>"
+
+msgid "You should encourage all volunteers to take the CoderDojo E-Learning modules."
+msgstr "You should encourage all volunteers to take the CoderDojo E-Learning modules."
+
+msgid "Venue Details"
+msgstr "Venue Details"
+
+msgid "Sign the Charter"
+msgstr "Sign the Charter"
+
+msgid "Review your application"
+msgstr "Review your application"
+
+msgid "Congratulations! Your Dojo application is being reviewed by a member of the CoderDojo Foundation team."
+msgstr "Congratulations! Your Dojo application is being reviewed by a member of the CoderDojo Foundation team."
+
+msgid "We will will respond to you within 48 hours, so hang tight while we check the information you have submitted."
+msgstr "We will will respond to you within 48 hours, so hang tight while we check the information you have submitted."
+
+msgid "Something went wrong while submitting your application, please contact support"
+msgstr "Something went wrong while submitting your application, please contact support"
+
+msgid "Prepare yourself"
+msgstr "Prepare yourself"
+
+msgid "While waiting for your verification, familiarise yourself with our content!"
+msgstr "While waiting for your verification, familiarise yourself with our content!"
+
+msgid "You didn't finish your application"
+msgstr "You didn't finish your application"
+
+msgid "Contact us if you have a question"
+msgstr "Contact us if you have a question"
+
+msgid "Last updated on {{ date }}"
+msgstr "Last updated on {{ date }}"
+
+msgid "Started on {{ date }}"
+msgstr "Started on {{ date }}"
+
+msgid "Expect a 2 day response time"
```

Comment:
Same point as 48 hours above. Wording should also be consistent. Additionally, small numbers should be written as words. I guess 'two working days' would work best? /cc @rosalanghammer